terra-jupyter-gatk-ovtf/tests/smoke_test.py

```diff
@@ -0,0 +1,14 @@
+"""Brief smoke tests for terra-jupyter-gatk-ovtf.
+
+Minimal tests simply import packages and run 'hello world' code to determine whether any exceptions
+are thrown.
+
+TODO(all): Add additional tests and/or tests with particular assertions, as we encounter Python package version
+incompatibilities not currently detected by these tests.
+"""
+
+import os
+import pytest
+
+def test_ovtf():
+  # TODO
```

Comment:
Can add `pytest`cases here. The GitHub action will execute these and report on failures.